backend/server/Cargo.toml

```diff
@@ -15,9 +15,11 @@ figment = { version = "0.10.19", features = ["env", "toml"] }
 http = "1.3.1"
 jsonwebtoken = "9.3.0"
 octocrab = "0.41.2"
+once_cell = "1.21.3"
```

Comment:
We have https://doc.rust-lang.org/stable/std/cell/struct.OnceCell.html now :)

Reply:
nice

Reply:
```
error[E0277]: `std::cell::OnceCell<Pool<Sqlite>>` cannot be shared between threads safely
 --> server/src/db/mod.rs:5:20
  |
5 | static SQLITEPOOL: OnceCell<SqlitePool> = OnceCell::new();
  |                    ^^^^^^^^^^^^^^^^^^^^ `std::cell::OnceCell<Pool<Sqlite>>` cannot be shared between threads safely
  |
  = help: the trait `Sync` is not implemented for `std::cell::OnceCell<Pool<Sqlite>>`
  = note: if you want to do aliasing and mutation between multiple threads, use `std::sync::OnceLock` instead
  = note: shared static variables must have a type that implements `Sync`
```
using `std::sync::OnceLock` for now